docs/mdbook/default.nix

```
@@ -204,9 +206,7 @@ with lib; let
             then "cp ${mkMDDoc opts.index.options} ${path}"
             else ''
               {
-              cat <<EOF
-              ${opts.index.moduleDoc}
-              EOF
+              cat ${moduleDocMD}
```

You can even inline it as so (the filename doesn't matter indeed).
```suggestion
              cat ${pkgs.writeText "module-doc" opts.index.moduleDoc}
```

---

Done :+1: